kiwi/runtime_checker.py

```diff
@@ -853,3 +934,33 @@ def check_image_type_unique(self):
                         type_export.getvalue()
                     )
                 )
+
+    @staticmethod
+    def _get_dracut_module_version_from_pdb(
+        package_manager, package_name, root_dir
+    ):
+        tool_for_package_manager = {
```

Comment:
I did it that way because the tool for debian is dpkg in get_default_packager_tool and I need dpkg-query. But you are right, it's better to handle that one exception instead of re-inventing the wheel. Will change it

Reply:
done now